app/services/data/export/write-stream-to-file.service.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
+
+/**
+ * Export converted data to a temporary file
+ * @module WriteStreamToFileService
```

Comment:
The name `WriteStreamToFileService` isn't quite correct any more; we're writing some headers and _then_ a stream to a file. So I wonder if `WriteTableToFileService` is more accurate? Alternatively, it's possible to get the table headers from the db as a stream, and then we change `ExportTableService` to call `WriteStreamToFileService` twice -- once with the header stream, and once with the rows stream. It should be fairly straightforward to do and might be a little cleaner overall, but since what we've got does work then I'm not sure if it's worth making the changes?

Reply:
I think I prefer the name change to writeTableToFileService rather then calling the stream twice :)